config-linux.md

```diff
@@ -185,6 +185,9 @@ If the value is specified, the runtime MUST consistently attach to the same plac
 If the value is not specified, the runtime MAY define the default cgroups path.
 Runtimes MAY consider certain `cgroupsPath` values to be invalid, and MUST generate an error if this is the case.
 
+Runtimes MAY reject a cgroup which it deems unfit; in particular, a frozen
+cgroup or, for a new container, a non-empty cgroup.
```

Comment:
"reject a cgroup" needs to be worded more explicitly because it's a bit ambigous what it means. I guess you'd want to say something more like:

> Runtimes MAY refuse to create or start a container which has been configured to use a `cgroupsPath` which it considers would not be fit for purpose. Examples include a frozen cgroup or (for a new container) a non-empty cgroup. The reason for this is that such cgroup settings could result in container operations on one container inadvertently affecting other containers in a way that users may not anticipate or understand.

There is a slight concern that technically this permission would allow you to create a runtime-spec-compliant runtime which simply refuses to start *any* containers but I guess that's a purely theoretical concern.

Reply:
> which has been configured to use a cgroupsPath

This is not entirely true. Even when `cgroupsPath` is not set, some default cgroup path is used, and that path is also checked against being empty and not frozen.

So, maybe this belongs to a different part of the spec. Let me take a look...

Reply:
Changed, PTAL